netbout-web/src/test/java/com/netbout/rest/TkPreviewTest.java

```diff
@@ -77,7 +77,7 @@ public void previewMessageBeforePost() throws Exception {
                 )
             ).printBody(),
             Matchers.allOf(
-                Matchers.containsString("<p>Here is a paragraph.</p>"),
+                Matchers.containsString("<p>Here is a paragraph.<br  /></p>"),
```

Comment:
@dmzaytsev this seems fragile to me (in case Txtmark changes and decides to render `<br/>` slightly differently). Maybe we can use [XhtmlMatchers](http://matchers.jcabi.com/xhtml-matchers.html)?